### PR TITLE
Add types for jexcel

### PR DESCRIPTION
**Pull request by arx-8**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/jexcel/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,895 @@
...
+    // tslint:disable-next-line ban-types
+    touchEndControls: Function;
+    // tslint:disable-next-line ban-types
+    touchStartControls: Function;
+    // tslint:disable-next-line ban-types
+    validLetter: Function;
+}
+
+declare const jexcel: JExcel;
+export default jexcel;
```
Comment (armanio123) [inline types/jexcel/index.d.ts:895]: I didn't saw a default export. But instead I did see on the code that this library looks like it uses UMD. If that's the case you need to use an `export as namespace` syntax.

See: https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html for a sample template.

**Review by armanio123 — CHANGES_REQUESTED** [review]

File: types/jexcel/index.d.ts (hunk 1 above)
Comment (arx-8) [reply types/jexcel/index.d.ts]: @armanio123 
Thank you for the review!
Until now, I didn't understand UMD well.

I correct the definition as UMD.
Could you check the src again?

File: types/jexcel/tslint.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+      "export-just-namespace": false
```
Comment (armanio123) [inline types/jexcel/tslint.json:4]: Could you enable this rule?

**Review by armanio123 — CHANGES_REQUESTED** [review]

File: types/jexcel/tslint.json (hunk 2 above)
Comment (arx-8) [reply types/jexcel/tslint.json]: Thank you for the review.
I'm sorry for the my mistake...
Could you check the src again?

**Review by armanio123 — APPROVED** [review]
Thanks for addressing all comments.

**Pull request merged**